src/store/login/index.js

```diff
@@ -55,9 +56,13 @@ export const setJwtToken = createAction(
   jwtToken => jwtToken
 );
 
+export const readJwtToken = createAction(
+  READ_JWT_TOKEN
+);
+
 export const setAuthError = createAction(
   SET_AUTH_ERROR,
-  error => error
+  (error = {}) => error
```

Comment:
I would put this default value into the reducer insted of the action, the action should be pure as posisble.

Reply:
sure, moving it to the reducer